docs/install/1-install-server.rst

```diff
@@ -433,11 +413,10 @@ ssh and if the remote agent is used.
 
     sudo chown inmanta:inmanta /var/lib/inmanta/.ssh/config
 
-3. Add the public key to any git repositories and save it to include in configuration models that require remote agents.
```

Comment:
Why drop 3? The git repository part is still relevant, right?

Reply:
Aah, do I understand correctly that this is to set up repos where developers would store .cf files. This has nothing to do with v1 modules, right ?

Reply:
This is required for two reasons:

* For cloning the Inmanta project. This is always a git repo.
* For cloning V1 modules (if used).